app/views/articles/show.html.erb

```diff
@@ -9,10 +9,10 @@
       <strong>Text:</strong>
       <%= @article.text %>
     </p>
-    <% if can? :update, @article %>
+    <% if current_user.has_role? :admin or current_user.has_role? :manager or current_user.has_role? :moderator %>
```

Comment:
write a method like and use these methods where ever u want
current_user.admin?
current_user.manager?
current_user.moderator?

Reply:
have created methods as you mentioned above, and use those methods by invoking using their objects (class variables)